pkg/link-controllers/nodecidr/adapter.go

```diff
@@ -103,7 +103,7 @@ func (c *commonAdapter) OnUpdate(oldObj, newObj interface{}) {
 
 // OnDelete handles object delete event and push the object to queue.
 func (c *commonAdapter) OnDelete(obj interface{}) {
-	runtimeObj, ok := obj.(api.Node)
+	runtimeObj, ok := obj.(*corev1.Node)
 	if !ok {
```

Comment:
Is this bug just effect flannel? Why?

Reply:
good question，It effect the common adapter，flannel is the representative that use common adapter
I  have  modified the commit and pr name